controllers/AuthController.js

```diff
@@ -60,8 +60,57 @@ export default class AuthController {
         lastName: registeredUser.lastName,
         email: registeredUser.email,
         isVerified: registeredUser.isVerified,
-        verificationToken,
-      },
+        verificationToken
+      }
+    });
+  }
+
+  /**
+   *
+   * @static
+   * @param {object} req express request object
+   * @param {object} res express response object
+   * @returns {object} returns user data
+   * @memberof AuthController
+   */
+  static async userSignin(req, res) {
+    const { email, password } = req.body;
+
+    const userEmail = await models.User.findOne({ where: { email } });
```

Comment:
Looking at this variable name, does this request return a user's email?

Reply:
the variable `email` is used to store the user's email in the request body...

Reply:
is that what you meant?

Reply:
I meant `userEmail`. Does the database request return a user's email?

Reply:
Okay. thanks for pointing that out. would change the variable name. it actually returns  the 
user object